sdk/typescript/src/client/types/common.ts

```diff
@@ -4,3 +4,12 @@
 
 export type Order = 'ascending' | 'descending';
 export type Unsubscribe = () => Promise<boolean>;
+export enum TransactionKind {
```

Comment:
if we already have `IotaTransactionKind` in `sdk/typescript/src/client/types/generated.ts` I think this enum is redundant